edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/AdvanceManifestFilingTO.java

```diff
@@ -0,0 +1,22 @@
+package org.dcsa.edocumentation.transferobjects;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import lombok.Builder;
+import org.dcsa.edocumentation.transferobjects.validation.AdvanceManifestFilingValidation;
+
+@AdvanceManifestFilingValidation
```

Comment:
@preetamnpr This should have been on the model (rather than the TO) class.  (The related classes have to be moved to edocumentation-domain component)

Reply:
Nvm me. I confused myself